### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by asimpledeveloper**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# just-test
\ No newline at end of file
+# just-test
+1
```
Comment (alidehkhodaei) [inline README.md:2]: Remove this

File: README.md [hunk 2]
```diff
@@ -1 +1,2 @@
-# just-test
\ No newline at end of file
+# just-test
```
Comment (alidehkhodaei) [inline README.md:1]: Change this

**Review by alidehkhodaei — CHANGES_REQUESTED** [review]

**Review by alidehkhodaei — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1 +1,2 @@
-# just-test
```
Comment (alidehkhodaei) [inline README.md:1]: lklkl

**Review by alidehkhodaei — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (alidehkhodaei) [inline README.md:1]: hjhj

**Review by alidehkhodaei — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (asimpledeveloper) [inline README.md:1]: lklkl

Comment (alidehkhodaei) [inline README.md:1]: kjk

Comment (alidehkhodaei) [inline README.md:1]: kjkjk

Comment (alidehkhodaei) [inline README.md:1]: lkllllll

Comment (alidehkhodaei) [inline README.md:1]: \

**Review by alidehkhodaei — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (alidehkhodaei) [inline README.md:1]: lklkl

Comment (alidehkhodaei) [inline README.md:1]: lkkkkkkkkkkk

**Review by alidehkhodaei — APPROVED** [review]

**Pull request merged**